build/features.py

```diff
@@ -102,30 +104,38 @@ def configure(self, build, conf):
             build.env.Append(CCFLAGS='-pthread')
             build.env.Append(LINKFLAGS='-pthread')
 
-        elif build.platform_is_windows and not conf.CheckLib(['setupapi', 'libsetupapi']):
-            raise Exception('Did not find the setupapi library, exiting.')
-        elif build.platform_is_osx:
-            build.env.AppendUnique(FRAMEWORKS=['IOKit', 'CoreFoundation'])
+        else:
+            self.INTERNAL_LINK = True
+            build.env.Append(CPPPATH=[self.HIDAPI_INTERNAL_PATH])
+            if build.platform_is_windows and not conf.CheckLib(['setupapi', 'libsetupapi']):
+                raise Exception('Did not find the setupapi library, exiting.')
+            elif build.platform_is_osx:
+                build.env.AppendUnique(FRAMEWORKS=['IOKit', 'CoreFoundation'])
 
         build.env.Append(CPPDEFINES='__HID__')
+        if self.INTERNAL_LINK:
```

Comment:
Since you fully qualify all the hidapi.h includes, is adding the 'hidapi' subfolder to the path needed?